README.md

```diff
@@ -215,6 +215,35 @@ If you installed WIX before running `cmake` you should find the
 1. Navigate to the directory where you checked out the foundationdb repo.
 1. Run `make`.
 
+#### FreeBSD
+
+1. Check out this repo on your server.
+1. Install compile-time dependencies from ports.
+1. (Optional) Use tmpfs & ccache for significantly faster repeat builds
+1. (Optional) Install a [JDK](https://www.freshports.org/java/openjdk8/) for Java Bindings. FoundationDB currently builds with Java 8.
+1. Navigate to the directory where you checked out the foundationdb repo.
+1. Build from source.
+
+    ```shell
+    sudo pkg install -r FreeBSD \
+        shells/bash devel/cmake devel/ninja devel/ccache  \
+        lang/mono lang/python3 \
+        devel/boost-libs devel/libeio \
+        security/libressl
+    mkdir build && cd build
```

Comment:
There is already a directory called "build" :( , so this mkdir will probably fail

Reply:
fixed in next push